### ISSUE TITLE
IP Authenticated Proxies

### ISSUE THREAD

**Issue by mrosales11** [dzt/easy-proxy#18]
Any chance you all will be building a generator for IP authenticated proxies in a future release?

Thanks all!

**Comment by skram** [comment]
I have included this on my fork, and will soon submit a pull request. Thoughts on integration?

![screen shot 2017-12-29 at 8 57 29 am](https://user-images.githubusercontent.com/176396/34438702-ae2506a2-ec76-11e7-9bcf-4e80899bb72a.png)

**Comment by mrosales11** [comment]
That looks great!! Would the username and password still be generated?

Also, a little new to this (sorry) so went to your fork, but am assuming the IP authentication won’t be live until your request is approved?

Appreciate the patience and thanks, again integration looks great!

**Comment by skram** [comment]
Yes. Username/Password are still generated as you can see in the screenshot, difference is its not needed if coming from authenticated IP. It's not on my fork yet, but will be sometime today.

**Comment by mrosales11** [comment]
Awesome, will keep my eye out for when you add.

**Comment by mrosales11** [comment]
We’re you able to add?

**Comment by mrosales11** [comment]
@skram , if I download from your fork will it have the IP authenticate option?

Apologies for the follow up, new to gothub, but do appreciate your help. 

Thanks!

**Comment by kfinest** [comment]
We're you able to add IP AUTH?
